render.go

```diff
@@ -185,8 +184,8 @@ func (*RenderSystem) Type() string {
 	return "RenderSystem"
 }
 
-func (rs *RenderSystem) Priority() int {
-	return 1
+func (*RenderSystem) Priority() int {
+	return -10000 // some low value
```

Comment:
Well, I don't want to use the minimum value, just a value that's low enough that people can use the values between this one and the default one (`0` or `1` or so). 

I could use 
```go
const RenderSystemPriority = -10000
```
but I don't know if that would add anything? What do you think, @matiwinnetou @paked?